turbopy/diagnostics.py

```diff
@@ -109,6 +110,23 @@ def _write_buffer(self):
             np.savetxt(f, self._buffer, delimiter=",")
 
 
+
+class NetCDFOutputUtility(OutputUtility):
+    """Diagnostic output utility that writes to NetCDF files
+    """
+    def __init__(self, filename, diagnostic_size, **kwargs):
+        self._filename = filename
+        self._buffer = np.zeros(diagnostic_size)
+        self._buffer_index = 0
+
+
+utilities = {"stdout": PrintOutputUtility,
+             "csv": CSVOutputUtility,
+             "netcdf": NetCDFOutputUtility,
+            }
```

Comment:
Is this meant to be in the `NetCDFOutputUtility` class?

```suggestion
        utilities = {"stdout": PrintOutputUtility,
                     "csv": CSVOutputUtility,
                     "netcdf": NetCDFOutputUtility,
                    }
```

Reply:
No, this is meant to be a "module level" variable, giving a mapping from the short name (the key) to the utility class. I could probably use the dynamic factory base class (that PhysicsModule etc use), but I thought that was unnecessarily complicated.